x-pack/legacy/plugins/graph/public/components/venn_diagram/_index.scss

```diff
@@ -0,0 +1,9 @@
+.gphVenn__left {
```

Comment:
Don't forget that index files should not contain any selectors but imports only. Please make a `_venn_diagram.scss` file and move the contents on this file there and then import the new file here.